localstack-core/localstack/services/apigateway/router_asf.py

```diff
@@ -147,7 +147,7 @@ def register_routes(self) -> None:
             strict_slashes=True,
         )
 
-    def invoke_rest_api(self, request: Request, **url_params: Dict[str, str]) -> Response:
+    def invoke_rest_api(self, request: Request, **url_params: str) -> Response:
```

Comment:
Is this really related to this changeset? Looking at the access pattern to `url_params` this can't be a string (and I would expect the params to be a dict).

Reply:
Not exactly related, but the type warning by PyCharm was driving me a bit crazy, and I've looked into it and it seems for `*args` and `**kwargs` you only type the values and not the variable in itself: https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values

I feel like I had found a better PEP for this but cannot find it anymore...

Reply:
Today I Learned! Thanks! Make sense, but isn't really obvious to be honest 😛

Reply:
It really isn't 😅 I've been looking at this warning for more than a year and only looked it now so 😬 it didn't make sense to me either